src/server/services/wms/qgsmaprendererjobproxy.h

```diff
@@ -61,10 +62,21 @@ namespace QgsWms
        */
       QPainter *takePainter();
 
+      /**
+       * @brief Returns reported errors
+       * @return error list
+       */
```

Comment:
If there are docstrings, I think it would be good if they provide more information than available in the method signature alrady :)